app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PostsController < ApplicationController
+    def index
+      @posts=Post.all.first
```

Comment:
This wouldn't actually give you the most-recently created one since the Posts all have a randomly assigned `created_at`